2-mandatory/1-create-functions.js

```diff
@@ -44,7 +49,14 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+  const formattedPercentages = numbers.map((number) => {
+    if (number > 100) {
+      number = 100;
+    }
+    return (Math.round(number * 100) / 100).toFixed(2) + "%";
```

Comment:
```suggestion
    return (Math.round(number * 100) / 100)+ "%";
```

toFixed is not needed here, it will already round to 2 decimal places.